README.md

```diff
@@ -1 +1,268 @@
+      "first": "Livingston",
+      "last": "Knox"
+    },
+    "company": "PETICULAR",
+    "email": "livingston.knox@peticular.io",
+    "phone": "+1 (827) 484-2991",
+    "address": "687 Ditmars Street, Chicopee, American Samoa, 1572",
+    "about": "Do sit eu tempor commodo adipisicing aute fugiat est. Adipisicing sit nisi do nulla nulla incididunt occaecat cupidatat magna ut. Veniam velit consectetur do consequat cupidatat excepteur incididunt quis eiusmod laborum. Aliqua commodo eu id officia non consectetur voluptate in reprehenderit reprehenderit aliquip pariatur. Aliquip velit amet dolore id eu adipisicing. Adipisicing nulla pariatur duis consectetur aute dolore nostrud officia commodo sunt aute.",
+    "registered": "Tuesday, March 17, 2015 12:10 PM",
+    "latitude": "-76.468255",
+    "longitude": "162.152781",
+    "tags": [
+      7,
+      "irure"
+    ],
+    "range": [
+      0,
+      1,
+      2,
+      3,
+      4,
+      5,
+      6,
+      7,
+      8,
+      9
+    ],
+    "friends": [
+      3,
+      {
+        "id": 1,
+        "name": "Rachelle Decker"
+      }
+    ],
+    "greeting": "Hello, Livingston! You have 5 unread messages.",
+    "favoriteFruit": "apple"
+  },
+  {
+    "_id": "56b0022d812642fc00b83afe",
+    "index": 4,
+    "guid": "fd467e1a-f0d7-4be9-aeec-cb1b9ad74e67",
+    "isActive": false,
+    "balance": "$3,777.90",
+    "picture": "http://placehold.it/32x32",
+    "age": 32,
+    "eyeColor": "brown",
+    "name": {
+      "first": "Cain",
+      "last": "Whitfield"
+    },
+    "company": "SUPPORTAL",
+    "email": "cain.whitfield@supportal.tv",
+    "phone": "+1 (911) 519-3406",
+    "address": "464 Atlantic Avenue, Boomer, Oklahoma, 4010",
+    "about": "Sit deserunt veniam ullamco commodo qui veniam deserunt pariatur ut in ad consequat labore proident. Ex laboris eiusmod et nulla culpa non amet laboris fugiat est. Adipisicing est veniam cillum excepteur cillum proident aute. Nisi laborum irure sunt magna fugiat occaecat eiusmod dolor occaecat est pariatur magna est enim. Cillum ex excepteur ut cillum do aliquip proident veniam laboris nostrud dolore labore cillum velit. Quis velit amet voluptate nostrud aute id deserunt tempor ea exercitation.",
+    "registered": "Saturday, November 7, 2015 3:51 PM",
+    "latitude": "-36.291757",
+    "longitude": "97.311145",
+    "tags": [
+      7,
+      "irure"
+    ],
+    "range": [
+      0,
+      1,
+      2,
+      3,
+      4,
+      5,
+      6,
+      7,
+      8,
+      9
+    ],
+    "friends": [
+      3,
+      {
+        "id": 1,
+        "name": "Rachelle Decker"
+      }
+    ],
+    "greeting": "Hello, Cain! You have 9 unread messages.",
+    "favoriteFruit": "apple"
+  }
+]
+```
+
+### Extra Info
+
+The data is an array. In JavaScript/Node, arrays are objects with extra properties so the pattern `module.exports = {}` will work the same for arrays: `module.exports = []`.
+
+Arrays are 0-based so the second element will have an index of 1.
```

Comment:
Ah, oops -- I made a comment on the same line asking for more explanation in the other PR. I'd side with @vicfriedman on this one.